src/app/core/services/navigation/group-nav-tree.service.ts

```diff
@@ -23,13 +24,17 @@ export class GroupNavTreeService extends NavTreeService<GroupInfo> {
     super(currentContent);
   }
 
-  childrenNavData(): OperatorFunction<GroupInfo|undefined,NavTreeElement[]> {
+  childrenNavData(): OperatorFunction<GroupInfo|undefined,NavTreeElement[]|Error> {
```

Comment:
Note (no need for changes): Probably it would make sense to work with fetch state if error has to be returned. (fetching may be useful to display, even if not done for now)